packages/localization/src/Alert.js

```diff
@@ -37,8 +37,6 @@ type AlertOptions = {
 
 type AlertButtonStyle = 'default' | 'cancel' | 'destructive';
 
-type AlertType = 'default' | 'plain-text' | 'secure-text' | 'login-password';
```

Comment:
Why are we removing this?

Reply:
Using alert type was deprecated, and has been removed in v 0.59

Reply:
https://github.com/facebook/react-native/commit/e2bd7db